lib/main.dart

```diff
@@ -31,13 +40,19 @@ void main() async {
     const SystemUiOverlayStyle(statusBarColor: Colors.transparent),
   );
 
+  storage=Get.find<StorageService>();
   runApp(
-    const UltimateAlarmClockApp(),
+    UltimateAlarmClockApp(),
   );
 }
 
+initialConfig() async{
+  await Get.putAsync(() => StorageService().init());
+}
+
 class UltimateAlarmClockApp extends StatelessWidget {
-  const UltimateAlarmClockApp({super.key});
+  UltimateAlarmClockApp({super.key});
+  final storage = Get.find<StorageService>();
```

Comment:
If you're finding the storage service here, then why define it again outside the scope?

Reply:
its redundant, correcting it rn